### PR TITLE
Benchmarks

### PR DESCRIPTION
**Pull request by fsaintjacques**

### CODE REVIEW

File: ursabot/docker.py [hunk 1]
```diff
@@ -224,11 +224,10 @@ def apt(*packages):
     """Generates apt install command"""
     template = dedent("""
         apt update -y -q && \\
-        apt install -y -q \\
-        {} && \\
+        apt install -y -q {} && \\
```
Comment (fsaintjacques) [inline ursabot/docker.py]: @kszucs I'm not sure why I needed this to make it work.

File: ursabot/docker.py [hunk 2]
```diff
@@ -224,11 +224,10 @@ def apt(*packages):
...
         rm -rf /var/lib/apt/lists/*
     """)
-    args = indent(' \\\n'.join(packages), _tab)
+    args = indent(' '.join(packages), _tab)
```
Comment (fsaintjacques) [inline ursabot/docker.py]: And this.

File: ursabot/docker.py (hunk 1 above)
Comment (kszucs) [reply ursabot/docker.py]: These are just making the dockerfiles readable - shouldn't affect the image building.

**Comment by kszucs** [comment]
I'm working on a heavy undertaking right now #34, I try to finish it today. We should probably rebase this one on top of that.

**Pull request closed without merging**